assets/svg/icons/tick.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="11" height="9" viewBox="0 0 11 9" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M1 3.83836L4.10586 7L10 1" stroke="white" stroke-width="1.5"/>
```

Comment:
Icons should all use `currentColor` for their color (stroke or fill) rather than a hardcoded value.

Reply:
Good spot. I had actually made that change at one point but have inadvertently rolled it back.